.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!-- Provide a brief description of the PR -->
+
+#### Related PRs / Issues
+
+<!-- List any related issues here -->
+
+#### API Impact
+
+<!-- How would this affect the current API? Does this add, extend, deprecate, remove, or break any existing API? -->
+
+#### Verilog / AGFI Compatability
+
+<!-- Does this change any generated Verilog, or the simulator memory map?  -->
+
+#### Release Notes
+
+<!-- You'll copy text from here to the dev-to-master PR.  -->
+
+### Contributor Checklist
+
+- [ ] Did you add Scaladoc to every public function/method?
+- [ ] Did you add at least one test demonstrating the PR?
+- [ ] Did you delete any extraneous prints/debugging code?
+- [ ] Did you state the API impact?
+- [ ] Did you specify the Verilog generation impact?
+<!-- Do this if this PR changes verilog or breaks the default AGFIs -->
+- [ ] (If applicable) Did you regenerate and publicly share AGFIs?
+<!-- Do this if this PR is a bugfix that should be applied to master -->
+- [ ] (If applicable) Did you mark the PR as "Please Backport"?
```

Comment:
I guess right now "Please Backport" targets the `master` branch?

Reply:
mhmm